lib/matplotlib/tests/test_backends_interactive.py

```diff
@@ -67,6 +67,8 @@ def _get_testable_interactive_backends():
     "webagg.open_in_browser": False,
     "webagg.port_retries": 1,
 })
+if len(sys.argv) == 2:
```

Comment:
Not that it really matters, but technically
```suggestion
if len(sys.argv) >= 2:
```
Also, it would be good to document somewhere explicitly that the first parameter is the toolbar.

Reply:
I went for a more general solution of json-encoded additional rcParams (and an explanatory comment).